packages/visualizations/src/components/Table/Table.svelte

```diff
@@ -17,11 +18,13 @@
     const tableId = `table-${generateId()}`;
 
     $: ({ value: records } = data);
-    // FIXME: Eslint is in conflict with prettier
-    // eslint-disable-next-line object-curly-newline
-    $: ({ columns, title, subtitle, description, source, unstyled, locale } = options);
+    $: ({ columns, title, subtitle, description, source, unstyled, locale, pages } = options);
     $: defaultStyle = !unstyled;
     $: updateLocale(locale);
+    /* Preserves paginations controls positioning
+    min heigh of table + controls = max-height of row * (number of rows + 1 for headers)
+    */
+    $: minHeight = pages ? `${MAX_ROW_HEIGHT * (pages.recordsPerPage + 1)}px` : 'none';
```

Comment:
No we still have to submit that to Meg, but it's really annoying to have the pagination moving between pages too.

Reply:
We asked the question https://app.shortcut.com/opendatasoft/story/49140/design-how-the-table-and-pagination-react-when-number-of-items-changes-between-two-pages. Waiting for an answer...

In the meantime, I'm in favor of adding the less code possible.  I prefer to add a function or behavior later rather than delete one.